README.md

```diff
@@ -298,6 +306,24 @@ console.log("will raise an exception if not in script_hashes.yml!")
 <% end %>
 ```
 
+### Public Key Pins
+
+```
+config.hpkp = {
+  max_age: 60.days.to_i,   # max_age is a required parameter
+  include_subdomains: true, # whether or not to apply pins to subdomains
+  # Per the spec, SHA256 hashes are the only currently supported format.
+  pins: [
```

Comment:
This syntax feels a little awkward (could just be me). Given that it would be extremely rare for each pin to use their own hash type, it may make sense to just set it globally:

```
...
pin_hash: "sha256",
pins: ["pin1base64hash", "pin2base64hash"],
...
```

Or, just bake the hash type into the pin

```
pins: ["sha256:pin1base64hash", "sha256:pin2base64hash"],
```

I tend to prefer the first style, as there is no reason you can't/shouldn't use the same hash format for all pins.